app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -15,26 +15,37 @@
 <% cs_select_id =  using_hostgroups_page? ? :content_source_id : :content_source_id %>
 <% cs_select_name =  using_hostgroups_page? ? 'hostgroup[content_source_id]' : 'host[content_facet_attributes][content_source_id]' %>
 <% cs_select_attr = using_hostgroups_page? ? 'content_source' : 'content_facet.content_source' %>
+<% if !using_hostgroups_page? && !@host.content_source.present? && @host&.hostgroup&.content_source.present? && cv_lce_disabled? %>
+  <% content_source_inherited = true %>
+<% end %>
```

Comment:
Seems this may be cleaner if it were a method in `host_and_hostgroups_helper.rb`.

```rb
def content_source_inherited?(host)
  !using_hostgroups_page? && !@host.content_source.present? && @host&.hostgroup&.content_source.present? && cv_lce_disabled?
end
```

Then you could just call it here like
```rb
content_source_inherited?(@host)
```